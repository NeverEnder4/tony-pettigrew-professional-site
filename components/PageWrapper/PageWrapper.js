import Head from 'next/head';

import './PageWrapper.scss';

const PageWrapper = ({ children }) => {
  return (
    <div>
      <Head>
        <meta charSet="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta httpEquiv="X-UA-Compatible" content="ie=edge" />

        <title>
          Tony Pettigrew | Web Developer | UX/UI Designer | Creative Solutions
          Expert
        </title>
        <link
          href="https://fonts.googleapis.com/css?family=Rubik:400,500,700,900"
          rel="stylesheet"
        />
      </Head>
      <div className="wrapper">{children}</div>
    </div>
  );
};

export default PageWrapper;
