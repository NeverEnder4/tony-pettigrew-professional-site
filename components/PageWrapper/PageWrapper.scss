@import '../../scss/global/global.scss';

.wrapper {
  padding: 6.8rem 10px 0px 10px;
  @media (min-width: 1280px) {
    padding: 12.3rem 20px 0px 20px;
  }
}
