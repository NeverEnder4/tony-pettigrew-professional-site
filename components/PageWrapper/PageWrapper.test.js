import renderer from 'react-test-renderer';
import PageWrapper from './PageWrapper';

it('renders children', () => {
  expect(
    renderer
      .create(
        <PageWrapper>
          <div>Test</div>
        </PageWrapper>,
      )
      .toJSON(),
  ).toMatchSnapshot();
});
