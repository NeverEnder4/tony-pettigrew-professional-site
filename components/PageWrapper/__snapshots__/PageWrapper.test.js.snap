// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders children 1`] = `
<div>
  <div
    className="wrapper"
  >
    <div>
      Test
    </div>
  </div>
</div>
`;
